bin/database.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+import json
+import sys
+from api import config
+
+CURRENT_DATABASE_VERSION = 1 # An int that is bumped when a new schema change is made
+
+def get_db_version():
+
+    version = config.get_version()
+    if version is None or version.get('database', None) is None:
+        return 0
+    else:
+        return version.get('database')
+
+
+def confirm_schema_match():
+    """
+    Checks version of database schema
+
+    Returns (0)  if DB schema version matches requirements.
+    Returns (42) if DB schema version does not match
+                 requirements and can be upgraded.
+    Returns (43) if DB schema version does not match
+                 requirements and cannot be upgraded,
+                 perhaps because code is at lower version
+                 than the DB schema version.
+    """
+
+    db_version = get_db_version()
+    if not isinstance(db_version, int) or db_version > CURRENT_DATABASE_VERSION:
+        sys.exit(43) 
+    elif db_version < CURRENT_DATABASE_VERSION:
+        sys.exit(42)
```

Comment:
In both of these first two cases, please print some details to stderr.
In particular, we should know if the script failed because db_version wasn't integer.

```python
import sys
# ...
sys.stderr.write('spam\n')
```